contracts/prize-strategy/PeriodicPrizeStrategy.sol

```diff
@@ -156,6 +166,13 @@ abstract contract PeriodicPrizeStrategy is Initializable,
     // 1 hour timeout
     _setRngRequestTimeout(3600);
 
+    emit Initialized( _trustedForwarder,
+     _prizePeriodStart,
+     _prizePeriodSeconds,
+     _prizePool,
+     _ticket,
+     _sponsorship,
+     _rng);
```

Comment:
Tabbing is wrong here: args are only 1 space and there is a random space before _trustedForwarder.

Keep it tight & consistent!